eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -15,6 +15,7 @@ parameters:
   ScriptDirectory: eng/common/scripts
   GHReviewersVariable: '' 
   GHTeamReviewersVariable: ''
+  GHAssignessVariable: ''
```

Comment:
As we talked are you planning to change these from Variable names to value? Or do you plan to do that as part of another change? I would at least do the new one you are adding as a value instead of a variable.

Reply:
I will do that in another PR. As far as this is concerned just passing the regular variable is sufficient.

Reply:
I'm Ok with that but it might add more debt and make it harder to change if someone takes a dependency on this.